src/components/Avatar/index.tsx

```diff
@@ -57,7 +59,7 @@ const AvatarComponent: FC<AvatarProps> = ({
               'p-1': bordered,
             })}
             src={img}
-            alt="Rounded avatar"
+            alt={alt}
```

Comment:
Should we have a default alt value if none is specified?

Reply:
My logic is to default to "" which browsers interpret as a [decorative image](https://www.w3.org/WAI/tutorials/images/decorative/).

The difference is that screen readers will skip over it rather than reading aloud, "Rounded avatar".

IMO, it would be a good idea to not provide an alt text unless the developer has decided that image should have one, and what it should be.

But that is my opinion.

For example, if I use the Avatar in context -- like, next to the user's name whose Avatar it is -- I would probably see no need to have an alt text. It would just be redundant.

On the other hand, if I use the Avatar as a trigger for a Dropdown, it would be nice to provide alt text, which would provide the screen reader context for the Dropdown.

Reply:
Fair enough, I didn't see the `''` default :sweat_smile: